driver-core/src/main/java/com/datastax/driver/core/ReplicationStategy.java

```diff
@@ -55,6 +56,7 @@ static ReplicationStrategy create(Map<String, String> replicationOptions) {
             }
         } catch (NumberFormatException e) {
             // Cassandra wouldn't let that pass in the first place so this really should never happen
+            logger.error("Failed to parse replication options: " + replicationOptions, e);
```

Comment:
The advantage of removing this catch block is that it removes some dead code and increases our test coverage. But I don't mind leaving it neither, so I'd suggest that we wait for some more input from others in the team: @olim7t, @newkek, @tolbertam, what are your thoughts?